testing/chain.go

```diff
@@ -150,7 +148,6 @@ func NewTestChainWithValSet(tb testing.TB, coord *Coordinator, chainID string, v
 		ChainID:        chainID,
 		App:            app,
 		ProposedHeader: header,
-		QueryServer:    app.GetIBCKeeper(),
```

Comment:
Is there any reason why you didn't want to keep this with `keeper.NewQueryServer(app.GetIBCKeeper().ChannelKeeper)`?

Reply:
This query server is the interface that was previously defined in `core/types` which I removed in this PR.
It was the union of all core query servers.
```golang
type QueryServer interface {
	clienttypes.QueryServer
	connectiontypes.QueryServer
	channeltypes.QueryServer
}
```

I'm not sure this provides a tonne of value and each submodule query server can be created on demand since the sub keepers are public on the ibc core keeper.

If folks feel strongly about keeping this type for testing purposes then I would suggest moving this to the `testing` pkg, which would still require creating a struct to passthrough to each submodule query server. Similar to what was removed in `core/keeper/grpc_query.go` in this PR.

Reply:
Sounds good to me. I don't feel strongly about it tbh, was mostly curious about the thinking, but it makes sense! 👍

Reply:
IIRC the query server was added to the testing chain struct for testing grpcs. Given that the individual query server is created as necessary, I agree it makes sense to remove. It also didn't contain all the queries available on the chain (non core ibc queries) so it might have led to confusion